docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  integration_tests:
```

Comment:
This is not just building the integration tests part, right? Could it be used to build the entire gsy-e?

Reply:
This is actually building a working gsy-e image, which in particular will be used to test the sdks. That's the reason it is named as integration_tests.
However, the same exact image could be used to run the exchange for a different purpose.

Reply:
So I would say it does serve to build the whole gsy-e.

However, as was mentioned on other comments, and after working on the comments left on GBIT I realized that adding this compose file on gsy-e is not really necessary.

Reply:
I would rename this service `gsy-e`, since it can be used for multiple purposes (not just integration tests).

Reply:
I saw your comment as well on GBIT, which has been replied to.